### ISSUE TITLE
Improve contrast in console (dark text on black console)

### ISSUE THREAD

**Issue by bender2k14** [dotnet/BenchmarkDotNet#1336]
When I run `BenchmarkDotNet.Samples.FSharp`, here is what the console looks like.
![2019-12-19_10-59-11_10_BenchmarkDotNet Samples FSharp](https://user-images.githubusercontent.com/34664007/71192956-e00d8f80-224e-11ea-9246-11c975842730.png)

The text is a dark color on a black background.  I am having difficulty seeing the text.

Can this contrast be improved?

**Comment by adamsitnik** [comment]
Hello @bender2k14 

> Can this contrast be improved?

Yes, but it would require your contribution.

The colors are defined [here](https://github.com/dotnet/BenchmarkDotNet/blob/3223c94a92050147a02482a3810860703f0c5171/src/BenchmarkDotNet/Loggers/ConsoleLogger.cs#L64-L75) Please feel free to experiment with them and send a PR.

Thanks,
Adam

**Comment by bender2k14** [comment]
Hello @adamsitnik, thanks for your response.

If you would prefer to reinvent logging (instead taking on dependency for logging), then I recommend copying how [Serilog.Sinks.Console](https://github.com/serilog/serilog-sinks-console#themes) does it.  Specifically, the coloring that you are currently using is fine but only apply it to the log type (which implies that it is also displayed).

What do you think?

**Comment by adamsitnik** [comment]
Do you mean using the same colors as Serilog?

![obraz](https://user-images.githubusercontent.com/6011991/77436183-779cf400-6de3-11ea-99c3-f44df402e247.png)

I am fine with that. @AndreyAkinshin what do you think?

**Comment by AndreyAkinshin** [comment]
@adamsitnik so, do I understand correctly that we are going to replace
```cs
{ LogKind.Help, ConsoleColor.DarkGreen },
```
with
```cs
{ LogKind.Help, ConsoleColor.White },
```
?

**Comment by adamsitnik** [comment]
@AndreyAkinshin yes, but we might also want to set bg color to red for error and yellow text color for warning

**Comment by bender2k14** [comment]
> Do you mean using the same colors as Serilog?

I don't care about the exact colors used.  I am suggesting that the text to be log should be white so that the contrast with the black background is high.

> do I understand correctly that we are going to replace...

No.  Sorry that I have been unclear.  I created PR #1404 that makes the kind of change that I am taking about.  It also includes a screenshot of the new behavior.

**Comment by adamsitnik** [comment]
I've marked the issue as `up-for-grabs`. Whoever is willing to work on this, please experiment with the color scheme that is defined here; https://github.com/dotnet/BenchmarkDotNet/blob/3223c94a92050147a02482a3810860703f0c5171/src/BenchmarkDotNet/Loggers/ConsoleLogger.cs#L64-L75

and find a better one ;)
